app/src/main/java/com/duckduckgo/app/ui/editbookmark/EditBookmarkDialogFragment.java

```diff
@@ -100,6 +101,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
                     @Override
                     public void onClick(DialogInterface dialog, int which) {
                         onSaveButtonClick();
+                        Toast.makeText(getContext(), "Bookmark Saved", Toast.LENGTH_SHORT).show();
```

Comment:
Yeah! I totally forgot that hardcoded string. Please have a look. Just pushed suggested changes.